FinniversKit/Sources/Fullscreen/FrontPage/FrontPageView.swift

```diff
@@ -388,10 +360,14 @@ extension FrontPageView: MarketsViewDelegate {
 
 // MARK: - FrontPageShelfDatasource
 extension FrontPageView: FrontPageShelfViewDataSource {
-    public func frontPageShelfView(_ frontPageShelfView: FrontPageShelfView, titleForSectionAt index: IndexPath) -> String {
+    public func frontPageShelfView(_ frontPageShelfView: FrontPageShelfView, titleForSectionAt index: Int) -> String {
         shelfViewModel?.titleForSection(at: index) ?? ""
     }
     
+    public func frontPageShelfView(_ frontPageShelfView: FrontPageShelfView, titleForButtonForSectionAt index: Int) -> String {
+        return "Se alle"
```

Comment:
This String should be added as part of `FrontPageShelfViewModel` instead, so it can be localized in the app 😄